### PR TITLE
simplify-installation

### PR DESCRIPTION
**Pull request by billpalombi**
Simplify installation page by removing unnecessary info, combining duplicative info, and reordering info

### CODE REVIEW

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ search:
 
 Prefect requires Python 3.8 or newer.
 
-Python 3.12 support is experimental, as not all dependencies support Python 3.12 yet. If you encounter any errors, please [open an issue](https://github.com/PrefectHQ/prefect/issues/new?assignees=&labels=needs%3Atriage%2Cbug&projects=&template=1_general_bug_report.yaml).
+Python 3.12 support is experimental, as not all of Prefect's dependencies support Python 3.12 yet.
```
[the remark this answers is not in the document]
Comment (discdiver) [reply docs/getting-started/installation.md:20]: lol - opened another PR to remove this line, simultaneous to yours.

[the remark this answers is not in the document]
Comment (billpalombi) [reply docs/getting-started/installation.md:20]: Saw that. I'll merge that change in.

[the remark this answers is not in the document]
Comment (discdiver) [reply docs/getting-started/installation.md:20]: Sorry, didn't mean to create a conflict.

[the remark this answers is not in the document]
Comment (billpalombi) [reply docs/getting-started/installation.md]: no problem

**Pull request merged**